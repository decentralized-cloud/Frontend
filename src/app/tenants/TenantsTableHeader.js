import React from 'react';
import TableHead from '@material-ui/core/TableHead';
import TableRow from '@material-ui/core/TableRow';
import TableCell from '@material-ui/core/TableCell';
import Checkbox from '@material-ui/core/Checkbox';

const TenantsTableHeader = () => (
  <TableHead>
    <TableRow>
      <TableCell padding="checkbox">
        <Checkbox checked={false} />
      </TableCell>
      <TableCell>Tenants Name</TableCell>
    </TableRow>
  </TableHead>
);

export default TenantsTableHeader;
