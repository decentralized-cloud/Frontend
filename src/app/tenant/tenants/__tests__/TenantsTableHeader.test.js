import React from 'react';
import renderer from 'react-test-renderer';

import TenantsTableHeader from '../TenantsTableHeader';

describe('<TenantView />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<TenantsTableHeader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
