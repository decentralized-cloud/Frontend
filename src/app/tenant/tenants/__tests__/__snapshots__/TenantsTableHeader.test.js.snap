// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TenantView /> renders correctly 1`] = `
<thead
  className="MuiTableHead-root"
>
  <tr
    className="MuiTableRow-root MuiTableRow-head"
  >
    <th
      aria-sort={null}
      className="MuiTableCell-root MuiTableCell-head MuiTableCell-paddingCheckbox"
      scope="col"
    >
      <span
        aria-disabled={false}
        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-25 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={null}
      >
        <span
          className="MuiIconButton-label"
        >
          <input
            checked={false}
            className="PrivateSwitchBase-input-28"
            data-indeterminate={false}
            onChange={[Function]}
            type="checkbox"
          />
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
      </span>
    </th>
    <th
      aria-sort={null}
      className="MuiTableCell-root MuiTableCell-head"
      scope="col"
    >
      Name
    </th>
  </tr>
</thead>
`;
